english/meaning/meaning.md

```diff
@@ -0,0 +1,5 @@
+You pause for a moment and stare up at the sky, ignoring the marshmallows or whatever junk is going on below you. Your moment of reflection ends suddenly when you realise you are in fact sitting at your computer reading this story. However, you are still confused as to what you just read and you go on to wonder where this story will take you next.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.